src/lib/components/Cytoscape.react.js

```diff
@@ -17,6 +17,7 @@ const cytoscape = require('cytoscape');
 const contextMenus = require('cytoscape-context-menus');
 
 // Clientside callback functions for CyLeaflet AIO component
+// eslint-disable-next-line no-unused-vars
 import * as cyleaflet_clientside from '../cyleaflet_clientside.js';
```

Comment:
```suggestion
import '../cyleaflet_clientside.js';
```
Rather than disable the warning, you can import the file but create no reference to it at all https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#import_a_module_for_its_side_effects_only

Reply:
Oh yeah! That's better! Thanks